index.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
You have two title tags here, I kindly suggest you use one title tag so as not to confuse the search engine and browser behaviour for SEO optimization.

Reply:
- [ ] Great work @Ayokunnumi1. Comment is clear, polite & explains why change is required 👏🏾